test/cljs/unit/ataru/hakija/application_validators_test.cljs

```diff
@@ -28,21 +29,39 @@
 (deftest ssn-validation
   (async done
          (asyncm/go
-           (doseq [ssn ssn/ssn-list]
-             (doseq [century-char ["A" "B" "C" "D" "E" "F" "U" "V" "W" "X" "Y" "-"]]
-               (let [ssn (str (:start ssn) century-char (:end ssn))]
-                 (is (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn" :value ssn})))
-                     (str "SSN " ssn " is not valid")))))
+           (with-redefs
+             [ataru.config/get-public-config (fn [keys] (get-in {:environment-name "sade"} keys))]
+             (doseq [ssn ssn/ssn-list]
+               (doseq [century-char ["A" "B" "C" "D" "E" "F" "U" "V" "W" "X" "Y" "-"]]
+                 (let [ssn (str (:start ssn) century-char (:end ssn))]
+                   (is (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn" :value ssn})))
+                       (str "SSN " ssn " is not valid")))))
+
+             (is (not (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn"})))))
+             (is (not (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn"})))))
+             (is (not (first (async/<! (validator/validate {:has-applied      (fn [_ _] (asyncm/go true))
+                                                            :validator        "ssn"
+                                                            :value            "020202A0202"
+                                                            :field-descriptor
+                                                            {:params {:can-submit-multiple-applications false
+                                                                      :haku-oid "dummy-haku-oid"}}}))))))
 
-           (is (not (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn"})))))
-           (is (not (first (async/<! (validator/validate {:has-applied has-never-applied :validator "ssn"})))))
-           (is (not (first (async/<! (validator/validate {:has-applied      (fn [_ _] (asyncm/go true))
-                                                          :validator        "ssn"
-                                                          :value            "020202A0202"
-                                                          :field-descriptor {:params {:can-submit-multiple-applications false
-                                                                                      :haku-oid                         "dummy-haku-oid"}}})))))
            (done))))
 
+(deftest test-ssn-validation
```

Comment:
olisko ssn-validation-test-hetu-does-not-work-in-production ?

Vois myös tehdä käänteisen testin jossa testihetu toimii muualla.